### DISCUSSION TITLE
New Mountain Duck user - so many issues

### DISCUSSION THREAD

**Discussion by eyemack** [iterate-ch/cyberduck#16616] [Q&A] [answered]
I bought Mountain Duck a day or so ago after a recommendation to use it for my OneDrive files.
I am having a whole load of issues.
1) I get dropped connections a lot.  Sometimes the whole Mac UI freezes.  Then I have to force a reboot.
2) very slow to show files in Finder.  Scrolling through folders with a lot of files is painfully slow.
3) very slow to generate previews of files, vidoes, etc. in Finder.
4) shows files as being on the disk when I have previously asked for them to be 'deleted on local drive'.
5) Mountain Duck does not load when starting my iMac.  I need to manually start it up.
6) files accessed via MD do not show in Mac finder 'Recents' on sidebar.
Any help on this would be appreciated.

**Comment by eyemack** [comment]
I opened a case with support and got a comprehensive response.

**Reply by muescha** [reply]
could you share the response so that it can help others?

**Comment by tatze96** [comment, marked as answer]
Summary of the points from the support case:
>1. I get dropped connections a lot. Sometimes the whole Mac UI freezes. Then I have to force a reboot.

This should be resolved or at least improved within Mountain Duck 4.17.0

>2. very slow to show files in Finder. Scrolling through folders with a lot of files is painfully slow.
>3. very slow to generate previews of files, vidoes, etc. in Finder.

Depending on the number of files present within the listing, this may be expected, especially if Finder simultaneously creates thumbnails and file previews.

>4. shows files as being on the disk when I have previously asked for them to be 'deleted on local drive'.

Using the option *Delete on Local Disk* doesn't prevent Mountain Duck from downloading the corresponding items into the cache again, for example, when Finder accesses them to create thumbnails and file previews.

Additionally, the *Delete on Local Disk* operation gets added to the synchronization queue, and the items get removed from the cache once the synchronization process reaches the point within the queue. So, the files may not be removed yet from the cache if the synchronization process is still in progress.

>5. Mountain Duck does not load when starting my iMac. I need to manually start it up.

Please verify that you have the *login item* setting activated within the corresponding Mountain Duck preferences tab.

If you are using a macOS version newer than macOS 12, then you may run into a known issue where Mountain Duck doesn't get added to the list of *Login items* for which we haven't found a solution yet. You can work around the autostart issue by adding Mountain Duck manually to the *Login item* list within `System Settings → General → Login Items`.

>6. files accessed via MD do not show in Mac finder 'Recents' on sidebar.

This is currently expected behavior as the "Recents" section only includes items on the local disk, and Mountain Duck mounts the bookmarks as network drives.
